src/Kunstmaan/NodeSearchBundle/Configuration/NodePagesConfiguration.php

```diff
@@ -137,14 +136,23 @@ public function createIndex()
         );
 
         foreach ($this->locales as $locale) {
-            $localeAnalysis = clone($analysis);
-            $language = $this->analyzerLanguages[$locale]['analyzer'];
+            // Multilanguage check
+            if (count(preg_grep('/[a-z]{2}_?+[a-zA-Z]{2}/', [$locale])) > 0) {
+                $locale = strtolower($locale);
+            }
 
-            //build new index
+            // Build new index
             $index = $this->searchProvider->createIndex($this->indexName . '_' . $locale);
 
-            //create index with analysis
-            $this->setAnalysis($index, $localeAnalysis->setupLanguage($language));
+            if (array_key_exists($locale, $this->analyzerLanguages)) {
+                $localeAnalysis = clone $analysis;
+                $language = $this->analyzerLanguages[$locale]['analyzer'];
+
+                // Create index with analysis
+                $this->setAnalysis($index, $localeAnalysis->setupLanguage($language));
+            } else {
+                $index->create([]);
```

Comment:
I think the best course of action is to somehow show a warning in the command output. Otherwise it will not be clear for developers why their search might not be behaving as expected without manually checking the index metadata. If a simple message is shown in the console command output then at least users will immediately know that they forgot to define an analyzer language